src/DAT/tablestats.jl

```diff
@@ -180,12 +180,11 @@ function getStatOutAxes(tab, agg, ::Type{<:WeightedCovMatrix})
     s = varsym(agg)
     icube = findfirst(isequal(s), varn)
     ax = tab.dc.incubes[icube].axesSmall[1]
-    oldname = YAXArrays.Cubes.Axes.axname(ax)
+    oldname = DD.name(ax)
     coname = string("Co", oldname)
-    v = ax.values
-    axtype = axt(ax)
-    a1 = axtype(oldname, copy(v))
-    a2 = axtype(coname, copy(v))
+    v = ax.val
+    a1 = DD.Dim{oldname}(copy(v))
```

Comment:
I am not sure, but this should be rather name2dim otherwise this might change an X type to a Dim{:X} but I would have to look up the proper syntax for that.